### DISCUSSION TITLE
Erro when open

### DISCUSSION THREAD

**Discussion by arturcorumba** [ErrorFlynn/ytdlp-interface#35] [Q&A]
Anyone get this error when try to start?
![image](https://user-images.githubusercontent.com/93468720/213885692-c9e8b39c-ed9f-4c20-8714-247da46dc61e.png)

**Comment by ErrorFlynn** [comment]
Line 1 column 1 is the first character, which should be `{`, but isn't (maybe the file got corrupted somehow?). Try deleting the settings file (`ytdlp-interface.json`, should be in the same folder as the program).
